app/Console/Commands/syncLiveBibleIsHighlights.php

```diff
@@ -0,0 +1,132 @@
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $db_name = config('database.connections.livebibleis_users.database');
+
+        $from_date = $this->argument('date');
+        $from_date = Carbon::createFromFormat('Y-m-d', $from_date)->startOfDay();
+
+        $this->initColors();
+        $filesets = BibleFileset::with('bible')->get();
+        $this->dam_ids = [];
+        $books = Book::select('id_osis', 'id')->get()->pluck('id', 'id_osis')->toArray();
+
+        echo "\n" . Carbon::now() . ': liveBibleis to v4 highlights sync started.';
+        $chunk_size = config('settings.v2V4SyncChunkSize');
+        DB::connection($db_name)
+            ->table('highlight')
+            ->where('created', '>=', $from_date)
+            ->orderBy('id')
+            ->chunk($chunk_size, function ($highlights) use ($filesets, $books) {
+                // live bible is user and highlights
+                $user_ids = $highlights->pluck('user_id')->toArray();
+                $highlight_ids = $highlights->pluck('id')->toArray();
+                // v4 data
+                $v4_users = User::whereIn('id', $user_ids)->pluck('id');
+                $v4_highlights = Highlight::whereIn('id', $highlight_ids)->pluck('id');
+
+                $dam_ids = $highlights->pluck('dam_id')->reduce(function ($carry, $item) use ($filesets) {
+                    if (!isset($carry[$item])) {
+                        if (isset($this->dam_ids[$item])) {
+                            $carry[$item] = $this->dam_ids[$item];
+                            return $carry;
+                        }
+                        $fileset = getFilesetFromDamId($item, true, $filesets);
+                        if ($fileset) {
+                            $carry[$item] = $fileset;
+                            $this->dam_ids[$item] = $fileset;
+                        }
+                    }
+                    return $carry;
+                }, []);
+
+                $highlights = $highlights->filter(function ($highlight) use ($dam_ids, $books, $v4_users, $v4_highlights) {
+                    return validateV2Annotation($highlight, $dam_ids, $books, $v4_users, $v4_highlights, false);
+                });
+
+                $highlights = $highlights->map(function ($highlight) use ($v4_users, $books, $dam_ids) {
+                    return [
+                        'user_id'           => $v4_users[$highlight->user_id],
+                        'bible_id'          => $dam_ids[$highlight->dam_id]->bible->first()->id,
+                        'book_id'           => $books[$highlight->book_id],
+                        'chapter'           => $highlight->chapter_id,
+                        'verse_start'       => $highlight->verse_id,
+                        'verse_end'         => $highlight->verse_id,
+                        'highlight_start'   => 1,
```

Comment:
highlight_start and highlighted_words should be referenced from $highlight instead of hardcoded